integrationTests/relayers/ethToMultiversXWithChainSimulator_test.go

```diff
@@ -157,178 +162,467 @@ type keysHolder struct {
 	ethAddress common.Address
 }
 
-type testConfig struct {
-	isMintBurnOnMvX bool
-	isNativeOnMvx   bool
-	isMintBurnOnEth bool
-	isNativeOnEth   bool
+func TestRelayersShouldExecuteTransfers(t *testing.T) {
+	t.Run("ETH->MVX and back, ethNative = true, ethMintBurn = false, mvxNative = false, mvxMintBurn = true", testRelayersShouldExecuteTransfersEthToMVX(
+		argSimulatedSetup{
+			mvxHexIsMintBurn: hexTrue,
```

Comment:
I know the reason for having these hex values but for the sake of better test code understanding I would have kept as plain bools and do the conversions where needed. Of course, this is a personal choice.